_part5.1/random_date.py

```diff
@@ -0,0 +1,37 @@
+#=======================
+# אין לי וִנִגְרֶט - 5.1
+#=======================
+
+from datetime import datetime
+import random
+
+#----------------------------------------------------
```

Comment:
Documentation is excellent and important, use docstring in this format:


def my_function(arg1):
	"""
	Summary line.

	Extended description of function.

	Parameters:
	arg1 (int): Description of arg1

	Returns:
	int: Description of return value

	"""

	return arg1

print(my_function.__doc__)

if you want to read more https://www.geeksforgeeks.org/python-docstrings/